dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -396,7 +400,12 @@ public DOIStatus getDoiStatus() {
     // Warning: these 4 are forcing eager loaded version metadata
```

Comment:
Hmm. Make a comment and ticket, this should probably be done via query if we want to optimise 
ticket can cite  https://github.com/dockstore/dockstore/pull/4118/files as an example

Reply:
Whoops, missing period. 
Re-phrasing
Hmm. Make a comment and ticket, this should probably be done via query if we want to optimize. (To avoid these 4 forcing eager loaded version metadata). The solution suggested in the ticket will possibly be something like  https://github.com/dockstore/dockstore/pull/4118/files as an example which uses queries instead of loading` authors` as Java objects

Reply:
Thanks for the rephrase!
Opened #4206
Will see if I can at least move the author & email ones to use queries here...

Reply:
Looking into it more, the effort required for this probably makes sense to throw to the new ticket #4206